### PR TITLE
style fixed, checked by tool ngx-style

### PR DESCRIPTION
**Pull request by doujiang24**
`ngx-style` is the newly tool in this pr:
https://github.com/openresty/nginx-devel-utils/pull/5

### CODE REVIEW

File: src/ngx_http_lua_socket_tcp.h [hunk 1]
```diff
@@ -31,8 +31,9 @@ typedef
         ngx_http_lua_socket_tcp_upstream_t *u, lua_State *L);
 
 
-typedef void (*ngx_http_lua_socket_tcp_upstream_handler_pt)(
-          ngx_http_request_t *r, ngx_http_lua_socket_tcp_upstream_t *u);
+typedef
+    void (*ngx_http_lua_socket_tcp_upstream_handler_pt)(
+        ngx_http_request_t *r, ngx_http_lua_socket_tcp_upstream_t *u);
```
Comment (doujiang24) [inline src/ngx_http_lua_socket_tcp.h]: @agentzh I'm not sure if I did right here.

Comment (agentzh) [inline src/ngx_http_lua_socket_tcp.h]: @doujiang24 I think the last `(` should go with the first function parameter.

Comment (agentzh) [inline src/ngx_http_lua_socket_tcp.h]: @doujiang24 Maybe something like this?

```C
typedef void (*ngx_http_lua_socket_tcp_upstream_handler_pt)
    (ngx_http_request_t *r, ngx_http_lua_socket_tcp_upstream_t *u);
```

File: src/ngx_http_lua_socket_udp.h [hunk 2]
```diff
@@ -20,8 +20,9 @@ typedef
         ngx_http_lua_socket_udp_upstream_t *u, lua_State *L);
 
 
-typedef void (*ngx_http_lua_socket_udp_upstream_handler_pt)(
-          ngx_http_request_t *r, ngx_http_lua_socket_udp_upstream_t *u);
+typedef
+    void (*ngx_http_lua_socket_udp_upstream_handler_pt)(
+        ngx_http_request_t *r, ngx_http_lua_socket_udp_upstream_t *u);
```
Comment (agentzh) [inline src/ngx_http_lua_socket_udp.h]: Let's put the last `(` on the same line of its parameters as above :)

**Comment by agentzh** [comment]
@doujiang24 I have to say your coding style checking tool is really wonderful :) I am indeed much sloppier than I originally thought ;)

**Comment by agentzh** [comment]
@doujiang24 Other than those two function pointer typedefs mentioned above, it looks good to me :)

**Comment by doujiang24** [comment]
@agentzh Just fixed.

**Comment by agentzh** [comment]
@doujiang24 Patch applied to master. Thanks a lot!

**Pull request closed without merging**